### ISSUE TITLE
The requested URL was not found on this server.

### ISSUE THREAD

**Issue by rfrancocantero** [claudehohl/Stikked#548]
Hi,
I have a freshly installed Stikked environment.

I installed it in the directory

`/var/www/html/pastebin
`

the content of the directory is:

`
drwxr-xr-x 5 www-data www-data  4096 Feb 24 20:15 .

drwxr-xr-x 4 www-data www-data  4096 Feb 24 20:14 ..

-rw-r--r-- 1 www-data www-data  5844 Nov 24  2019 AUTHORS.md

-rw-r--r-- 1 www-data www-data  6982 Nov 24  2019 CC0

-rw-r--r-- 1 www-data www-data 11365 Nov 24  2019 README.md

drwxr-xr-x 3 www-data www-data  4096 Feb 24 20:14 doc

drwxr-xr-x 3 www-data www-data  4096 Feb 24 20:14 docker

-rw-r--r-- 1 www-data www-data   819 Nov 24  2019 docker-compose.yml

drwxr-xr-x 6 www-data www-data  4096 Feb 24 20:14 htdocs

`

My stikked.php

` 
*/
$config['site_name'] = 'Stikked';

/**
 * Base URL
 *
 * Set the base URL of Stikked. WITH trailing slash!
 *
 */
$config['base_url'] = 'http://domain.com/pastebin/htdocs/';

/**
`

So, the frontpage works. But, when clicking CREATE, or when clicking RECENT, TRENDING, API or ABOUT, it all gives a 404 error page not found. I can't get my head around it. Can someone help me please?

**Comment by The-Compiler** [comment]
Why the `/htdocs` in `base_url`?

**Comment by rfrancocantero** [comment]
It tried with / without but no luck. This was the latest version before posting it here.

**Comment by The-Compiler** [comment]
I'm not talking about the `/`, I'm talking about the `htdocs` part. Do you to to `http://domain.com/pastebin/htdocs/` to access your stikked instance?

**Comment by rfrancocantero** [comment]
Yes I do, but I also tried installing it in the root folder of var/www/html and changing the base url. Same result. Front page works, rest doesn't.

**Comment by tcreek** [comment]
This is NOTABUG.

Its your misconfiguration. You need to set a DocumentRoot = something like /var/www/html/pastebin/htdocs
Then set base URL to something like "http://mypaste.com"
